_posts/2016/2016-05-13-geecon.md

```diff
@@ -0,0 +1,86 @@
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Although I would not agree on all of his points it was a very critical view on the industry that really made me think.
+
+## What else
+
+Other talks had a lot of value for me because the topic was directly related to our work at epages:
+
+ **"Scaling Microservices at Gilt" - Adrian Trenaman**
+
+ Talk about the evolution of Gilt's architecture towards today's state with 300 microservices in production.
+
+ > <blockquote class="twitter-tweet" data-cards="hidden" data-lang="de"><p lang="en" dir="ltr">Today&#39;s slides! GeeCON scaling micro services at gilt by <a href="https://twitter.com/adrian_trenaman">@adrian_trenaman</a> <a href="https://twitter.com/GeeCON">@GeeCON</a> <a href="http://t.co/JUgu63XPGf">http://t.co/JUgu63XPGf</a> via <a href="https://twitter.com/SlideShare">@SlideShare</a> <a href="https://twitter.com/gilttech">@gilttech</a></p>&mdash; Adrian Trenaman (@adrian_trenaman) <a href="https://twitter.com/adrian_trenaman/status/642320016831041536">11. September 2015</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+ **"Kubernetes in Production" - Paul Bakker**
+
+Paul Bakker talked about running Kubernetes in production including an approach to implement blue-green deployments using the Kubernetes API.
+
+**"The Wix Microservice stack" - Tomer Gabel**
+
+Tomer Gabel talked about Wix's very pragmatic micoservice architecture that can do without container spphisticated schedulers and dynamic service discovery.
+
+## Conclusion
+
+The 900km trip from Hamburg to Kraków was really worth it and I enjoyed it a lot - not only because of the conference but also because Kraków has a lot to offer and is well worth a trip on its own.
+{% image blog/blog-geecon-1.jpg 75% left %} {% endimage %}
+So if you go there make sure you have a walk at the Vistula River, take a look at [Wawel Castle](http://www.wawel.krakow.pl/en/) and have a Kasztelan beer in the beautiful Stare Miasto - Kraków's historic old city centre - which is part of the [UNESCO World Heritage](http://whc.unesco.org/en/list/29).
+
+Was it all just fine? Of course not - as always there is room for improvement - the conference was really packed and either the venue is too small or they sold too many tickets. So rooms were always crowded and the floor always packed - which starts to be a little annoying after spending three days in such an environment.
+
+But apart from that I can only say thanks to the organisers - they put together a great conference that offers a good value and especially if you take into account that they are organising this besides their main job the result is pretty amazing.
```

Comment:
But apart from that, I can only say thanks to the organisers. They put together a great conference that offers a good value, especially, when taking into account, that they organised it besides their main job, the result is pretty amazing.